lib/Sema/CSSimplify.cpp

```diff
@@ -10993,6 +10999,10 @@ ConstraintSystem::SolutionKind ConstraintSystem::simplifyFixConstraint(
 
     return SolutionKind::Solved;
   }
+          
+  case FixKind::AllowWrappedValueMismatch: {
```

Comment:
Nit: Not important, but since there is not custom logic here this could just be added as extra case on [here](https://github.com/apple/swift/blob/37097c130ae45842ffbf23c8f74b080e46564ed3/lib/Sema/CSSimplify.cpp#L11036) :)